composer.json

```diff
@@ -22,6 +22,12 @@
         "mockery/mockery": "~0.9.1",
         "symfony/phpunit-bridge": "~3.3@dev"
     },
+    "suggest": {
+        "ext-openssl": "required for encrypted connections, dkim, smime, NTLMAuthenticator, ...",
+        "ext-bcmath": "required for NTLMAuthenticator",
+        "ext-mbstring": "mbstring or iconv required for multi byte support",
```

Comment:
I suggest that we require `symfony/polyfill-mbstring` instead.

Reply:
I guess a polyfill is not required.
I think there should be no real php7.x environment without extensions mbstring or iconv.
Also "suggest" only creates a notice if the extension is not installed and will be shown after `$ composer require "swiftmailer/swiftmailer:^6.0"`. I only added the "suggestions" to create a hint for misconfigured environments while using composer and not while runtime.

Reply:
For the record, I think that "suggest" are useless. Nobody reads them. And it's a one time thing.